syntax = "proto3";

package image_search;

service ImageSearch {
    rpc SearchImage (KeywordRequest) returns (ImageResponse) {}
}

message KeywordRequest {
    string keyword = 1;
}

message ImageResponse {
    bytes image_data = 1;
}